hadoop-hdds/tools/pom.xml

```diff
@@ -109,5 +109,11 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>hdds-test-utils</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>hdds-interface-admin</artifactId>
+      <version>${hdds.version}</version>
+    </dependency>
```

Comment:
`hdds-interface-admin` is already a transitive dependency via `hdds-server-framework`, so I don't think this is needed.